model/authorizations/v1/root_resource.model

```diff
@@ -36,6 +36,18 @@ resource Root {
 		target ResourceReview
 	}
 
+	// Reference to the resource that is used to submit Red Hat's Terms and Conditions
+	// for using OpenShift Dedicated and MOA review requests.
+	locator TermsReview {
+		target TermsReview
+	}
+
+	// Reference to the resource that is used to submit Red Hat's Terms and Conditions
+	// for using OpenShift Dedicated and MOA self-review requests.
```

Comment:
replace MOA -> Amazon Red Hat OpenShift everywhere.

LGTM otherwise

Reply:
Replaced. Would advise you to replace MOAEnabled to something else for the same reason